lib/grape/roar/extensions/relational_models/validations/active_record.rb

```diff
@@ -0,0 +1,71 @@
+module Grape
+  module Roar
+    module Extensions
+      module RelationalModels
+        module Validations
+          module ActiveRecord
+            include Validations::Misc
+
+            def belongs_to_valid?(relation)
+              relation = klass.reflections[relation]
+
+              return true if relation.is_a?(
+                ::ActiveRecord::Reflection::BelongsToReflection
+              )
+
+              invalid_relation(
+                ::ActiveRecord::Reflection::BelongsToReflection,
+                relation.class
+              )
+            end
+
+            # rubocop:disable Style/PredicateName
```

Comment:
Just run `rubocop -a` and `rubocop --auto-gen-config` and no need for any custom enables/disables.